tasks/db_deploy/migrations/migrate_versions_7_to_8/migrate_sql.py

```diff
@@ -35,19 +35,12 @@ def schema_versions_data_sql() -> text:  # pragma: no cover
     )
 
 
-def add_delete_file_to_files_table_sql() -> text:
+def add_delete_file_to_granules_table_sql() -> text:
```

Comment:
https://github.com/nasa/cumulus-orca/blob/0f9f8edcab0bc91b3cbbf127fd1cbbac3da2de82/tasks/db_deploy/migrations/migrate_versions_7_to_8/migrate_sql.py#L31 
this also needs update